QueueManager/QueueFileManager.cs

```diff
@@ -1,7 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
```

Comment:
When you create a new class in VS it puts in a few defaults. It's only in recent versions of VS that they grey them out to let you know when they aren't used.